scout/server/blueprints/variant/templates/variant/acmg.html

```diff
@@ -75,12 +78,22 @@ <h4>Evidence of {{ category }}</h4>
                         <label class="form-check-label" for="checkbox-{{ criterion_code }}"></label>
                       </div>
                     </div>
-                    <div id="comment-{{ criterion_code }}" class="{{ 'collapse' if not (comment or link) }} mt-2">
+                    <div id="comment-{{ criterion_code }}" class="{{ 'collapse' if not (comment or link or modifier) }} mt-2">
+                      <div class="row">
+                        <select {{ 'disabled' if evaluation }} id="modifier-{{ criterion_code }}" name="modifier-{{ criterion_code }}" class="form-control form-select">
+                          <option value="" {% if not modifier %}selected{% endif %}>Strength modifier...</option>
+                          {% for level in "Strong", "Moderate", "Supporting" %}
+                            {% if(level != evidence) %}
```

Comment:
Does very strong need a 'strong' modifier?

<img width="445" alt="image" src="https://user-images.githubusercontent.com/28093618/212006426-106c1175-2d45-4f6a-a3c3-a8c9a673f6c3.png">

Reply:
And stand alone as well, for the benign ones:

<img width="398" alt="image" src="https://user-images.githubusercontent.com/28093618/212006717-32de863a-0cf6-4f29-b636-5f0379729dd1.png">

Reply:
A right, they actually do - it serves as a downgrade in this case. Like for instance if you have a frame shift but it’s rather late in the gene though not in the very final part. Then perhaps Strong is better than Very Strong.

Conversely,  the scheme could use Stand Alone or Very Strong modifiers for other criteria but I’ve never seen it in any paper or the need for it in practice.